### PR TITLE
Moving psionics into a content package.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Want to see if this passes Travis. Also need to test it post-corppackage.

Depends on #41

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Would be good to get a review of this, but merge can wait till I actually test all the psionics changes.

File: nebula.dme [hunk 1]
```diff
@@ -3105,8 +3059,8 @@
 #include "maps\antag_spawn\wizard\wizard.dm"
 #include "maps\away\away_sites.dm"
 #include "maps\away_sites_testing\away_sites_testing_define.dm"
-#include "maps\bearcat\bearcat.dm"
 #include "maps\bearcat\bearcat_define.dm"
+#include "maps\example\example.dm"
```
Comment (comma) [inline nebula.dme]: hiss

File: code/modules/spells/spell_code.dm [hunk 2]
```diff
@@ -222,11 +222,6 @@ var/list/spells = typesof(/spell) //needed for the badmin verb for now
 		to_chat(user, "<span class='warning'>You shouldn't have this spell! Something's wrong.</span>")
 		return 0
 
-	var/spell_leech = user.disrupts_psionics()
```
Comment (comma) [inline code/modules/spells/spell_code.dm]: maybe do 'disrupts_spells()' with dunno nullrod by default, and override with psi stuff in pack?

File: nebula.dme (hunk 1 above)
Comment (MistakeNot4892) [reply nebula.dme]: Bearcat sucks!

**Comment by MistakeNot4892** [comment]
Tested, all works fine ticked and unticked.

**Review by comma — APPROVED** [review]

**Pull request merged**